packages/prisma/sql/getSelectedCalendarsToWatch.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
```

Comment:
Is this used?

Reply:
Not yet. But it is useful to have. Also once que upgrade prisma we can use it for typedRaw queries